tests/unit/test_chronos.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/python
+
+from __future__ import print_function
+import unittest
+import os
+import sys
+import requests
+import json
+from cli.chronos import Chronos
+from cli.marathon import Marathon
+from cli.framework import Framework
+from cli.appconfig import AppConfig
+from cli.utils import Utils
+from mockito import mock, when
+utils = Utils()
+
+
+class TestChronos(unittest.TestCase):
+    def setUp(self):
+        self.marathon = Marathon()
+        self.appconfig = AppConfig()
+
+    def test_image_name(self):
+        res = mock(requests.Response)
+        url = 'https://chronos-dev-roger.dal.moz.com/scheduler/jobs/search?name=application-platforms-app1'
```

Comment:
Since this repo is public, it would be good if we can load the Chronos FQDN using a config or command line arg